0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md

```diff
@@ -1,14 +1,72 @@
 ---
 title: The Connector to Connector protocol
-draft: 2
+draft: 3
 ---
 # Connector to Connector protocol
 
 ## Discovery
 
 Connectors discover their peers through out-of-band communication, or by looking at https://connector.land and contacting the administrator of another connector.
 
-Once peered, two connectors have a ledger between them; this is often a ledger with just two accounts, often administered collaboratively by the two connectors.
+Once peered, two connectors have a ledger between them; this is often a ledger with just two accounts.
```

Comment:
> this is often a ledger with just two accounts.

That raises more questions than it answers. What is a ledger with two accounts? Why is that the option that's often chosen?

Reply:
ok, that was a phrase that was already there before this PR, but I removed it.